### PR TITLE
added button on walls to be realesed

### PR DESCRIPTION
**Pull request by RoiPeleg**
@casey-collab please add

### CODE REVIEW

File: english/sleep/marshmallow.md [hunk 1]
```diff
@@ -40,3 +40,5 @@ Do you:
 [Sit down to meditate](sit-down-to-meditate/sit-down-to-meditate.md)
 
 [Wish for ice cream](./ice-cream/ice-cream.md)
+
+[look for something that will help you on the walls](lookwalls/lookwalls.md)
```
Comment (casey-collab) [inline english/sleep/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/lookwalls/lookwalls.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+you find a big red button.
```
Comment (casey-collab) [inline english/lookwalls/lookwalls.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RoiPeleg** [comment]
@casey-collab

File: english/lookwalls/lookwalls.md (hunk 2 above)
Comment (casey-collab) [inline english/lookwalls/lookwalls.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RoiPeleg** [comment]
@casey-collab

File: english/sleep/lookwalls/lookwalls.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+you find a big red button.
+do you press it ?
+
+[PRESS](/press/press.md)
```
Comment (casey-collab) [inline english/sleep/lookwalls/lookwalls.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/sleep/lookwalls/press/press.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+BOOOM!!
```
Comment (casey-collab) [inline english/sleep/lookwalls/press/press.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/lookwalls/lookwalls.md (hunk 2 above)
Comment (casey-collab) [inline english/lookwalls/lookwalls.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RoiPeleg** [comment]
@casey-collab

File: english/lookwalls/lookwalls.md (hunk 2 above)
Comment (casey-collab) [inline english/lookwalls/lookwalls.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RoiPeleg** [comment]
@casey-collab

File: english/lookwalls/press/press.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+BOOOM!!
```
Comment (casey-collab) [inline english/lookwalls/press/press.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RoiPeleg** [comment]
@casey-collab

**Comment by casey-collab** [comment]
It looks like the file `-w` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RoiPeleg** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**